src/main/java/ch/njol/skript/expressions/ExprVillagerType.java

```diff
@@ -0,0 +1,66 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Villager;
+import org.bukkit.entity.Villager.Type;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Villager Type")
+@Description("Represents the Type of a villager.")
```

Comment:
```suggestion
@Description("Represents the type of a villager. This usually represents the biome the villager is from.")
```
just for clarity. on first read, i didnt know what this signified